pkg/kubelet/container/container_gc.go

```diff
@@ -28,6 +28,9 @@ type ContainerGCPolicy struct {
 	// Minimum age at which a container can be garbage collected, zero for no limit.
 	MinAge time.Duration
 
+	// Maximum age at which a death container can survive, zero for no limit.
+	MaxSurvivalAge time.Duration
```

Comment:
Other configuration settings in these vein are using `less than zero` to mean no limit.

Reply:
Nit: `s/death/dead`.